resources/components/base/Suggestion.vue

```diff
@@ -75,6 +75,7 @@ module.exports = {
 
 	&:hover,
 	&:focus {
+		background-color: @background-color-framed--hover;
```

Comment:
This will need to be overridden for `mw-suggestion--confirmed` – as it is, the suggestion background is white on hover for confirmed suggestions, and I think it should remain `@background-color-primary`. This is certainly debatable, but I think it could be confusing for a user who clicks on a suggestion and doesn't move the mouse as it's toggled to the confirmed state.

Reply:
Additionally, now that we're transitioning another property, we should update the transition rule for `.mw-suggestion`. I'd recommend copying what's there for Button (I should have done that when writing this component in the first place...)

Reply:
Adding a separate mixin line for transition: background-color was the only way I could get this to work – please take a look and ensure this is correct.

Reply:
Unfortunately adding 2 lines means the first transition rule will be overridden by the second since the CSS will contain 2 transition rules. I'll push a change to copy the syntax I'm using for Button.